import/dproto/serialize.d

```diff
@@ -460,19 +467,19 @@ void writeProto(string T, R)(ref R r, BuffType!T src)
 }
 
 /// Ditto
-void writeProto(string T, R)(ref R r, const BuffType!T src)
+void writeProto(string T, R)(ref R r, const BuffType!T src) @trusted
 	if(isProtoOutputRange!R &&
 	  (T.msgType == "double".msgType || T.msgType == "float".msgType))
 {
 	r.put(src.nativeToLittleEndian!(BuffType!T)[]);
 }
 
 /// Ditto
-void writeProto(string T, R)(ref R r, const BuffType!T src)
+void writeProto(string T, R)(ref R r, const BuffType!T src) @trusted
```

Comment:
I don't feel that comfortable with `@trusted` here either. Better would be `@safe` which would require `R.put()` to be either safe or trusted. So it would be the users choice.
This could break compatibility to older code.